tests/Bot.Builder.Community.Adapters.Tests/TwitterAdapterTests.cs

```diff
@@ -78,17 +79,53 @@ public async Task SendActivitiesAsyncShouldReturnEmptyResponsesWithEmptyActiviti
             }
         }
 
+        [TestMethod]
+        public async Task SendActivitiesAsyncShouldReturnEmptyResponsesWithActivities()
```

Comment:
Is this name wrong? The method returns an array of responses when you send Message Activities.

Reply:
Thanks @ceciliaavila, I think we can call it "SendActivitiesAsyncShouldReturnResponseWithNullId"?